AccessibilitySnapshot/iOSSnapshotTestCase/Classes/FBSnapshotTestCase+Accessibility.swift

```diff
@@ -53,6 +55,25 @@ extension FBSnapshotTestCase {
             return
         }
 
+        // Store the layers of the text fields whose animations will be
+        // paused in order to resume them after the snapshot test.
+        var pausedTextFieldLayers = [CALayer]()
+        if forceCursorVisible {
+            // Show the cursor of all text fields.
+            view.recursiveForEach(viewType: UITextField.self) { textField in
```

Comment:
Realizing this isn't quite the behavior we want actually. Going to iterate on this a bit more.

Reply:
Hmm... I'm not sure exactly what this will do right now. I _think_ setting it to the converted `CACurrentMediaTime()` is going to pause the animation in its current position. Instead we probably want something like:

```swift
func pauseAnimation() -> (_ speed: Float, _ timeOffset: CFTimeInterval) {
    let originalValues = (speed, timeOffset)
    speed = 0
    timeOffset = 0
    return originalValues
}

func resumeAnimation(originalSpeed: Float, originalTimeOffset: CFTimeInterval) {
    timeOffset = originalTimeOffset
    speed = originalSpeed
}
```